app/src/main/java/eu/darken/myperm/permissions/core/known/APerm.kt

```diff
@@ -75,14 +79,6 @@ sealed class APerm constructor(val id: Permission.Id) {
         override val tags = setOf(ManifestDoc)
     }
 
-    object MANAGE_MEDIA : APerm("android.permission.WRITE_MEDIA_STORAGE") {
-        override val iconRes: Int = R.drawable.ic_manage_media_24
-        override val labelRes: Int = R.string.permission_write_media_storage_label
-        override val descriptionRes: Int = R.string.permission_write_media_storage_description
-        override val groupIds: Set<PermissionGroup.Id> = grpIds(APermGrp.Files)
-        override val tags = setOf(ManifestDoc)
-    }
-
```

Comment:
I think this just needs the correct id `android.permission.MANAGE_MEDIA`, or was this duplicate?